src/Microsoft.Tye.Core/ProjectReader.cs

```diff
@@ -219,10 +219,18 @@ private static void EvaluateProject(OutputContext output, ProjectServiceBuilder
             project.TargetFramework = targetFramework;
             output.WriteDebugLine($"Found target framework: {targetFramework}");
 
+            // TODO: Parse the name and version manually out of the TargetFramework field if it's non-null
+            project.TargetFrameworkName = projectInstance.GetPropertyValue("_ShortFrameworkIdentifier");
+            project.TargetFrameworkVersion = projectInstance.GetPropertyValue("_ShortFrameworkVersion") ?? projectInstance.GetPropertyValue("_TargetFrameworkVersionWithoutV");
+
```

Comment:
This really seems suspect :( do we know that 3.1 and 5.0 SDKS behavre the same way?

Reply:
Yes, they do, at least by default they do. I still think we should do the janky fallback to parsing the tfm.